app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
```

Comment:
- [ ] The user foreign key in the comments table is user_id, this is also the same with the ERD. Kindly address this code to reflect what is in the schema.
- [ ] Kindly add a function that will increase the counter of the comments when a user adds a comment to a post.

![foreign-key](https://github.com/danielochuba/duo-blog/assets/53124548/e52d316b-78eb-42fe-8e31-5b33fdf397c2)